src/api/app/components/request_decision_component.rb

```diff
@@ -52,4 +64,14 @@ def forward_allowed?
   def make_maintainer_of
     @action.target_project + ("/#{@action.target_package}" if @action.target_package)
   end
+
+  def combine_forwarding_targets
```

Comment:
There are multi action requests with dozens and dozens of targets...

Reply:
So, what about saying "and forwarding N requests" instead of explicitly printing the targets?

Reply:
better but does the number matter?